osu.Game/Screens/OnlinePlay/OnlinePlayComposite.cs

```diff
@@ -89,6 +89,9 @@ public partial class OnlinePlayComposite : CompositeDrawable
         [Resolved(typeof(Room))]
         protected Bindable<bool> AutoSkip { get; private set; }
 
+        [Resolved(typeof(Room))]
+        protected Bindable<bool> NoScoreMultiplier { get; private set; }
```

Comment:
The naming is all over the place here; I've seen `IsScoreMultiplierDisabled` in `ScoreInfo`, `DisableScoreMultiplier` in `PlayerConfiguration`, and `NoScoreMultiplier` here. Can we pick one variant and stick with it?